code/datums/diseases/advance/symptoms/shedding.dm

```diff
@@ -28,21 +28,25 @@ BONUS
 	..()
 	if(prob(SYMPTOM_ACTIVATION_PROB))
 		var/mob/living/M = A.affected_mob
+		var/mob/living/carbon/human/H = M
+
+		if(!ishuman(M))
+			return
```

Comment:
```dm
		if(!ishuman(A.affected_mob))
			return
		var/mob/living/carbon/human/H = A.affected_mob
```
you can just do this section like so, if you're going to typecast to `H` you don't even need `M` here as you can just istype the `A.affected_mob`. There is a to_chat below that uses `M` which could be replaced with `H`